src/plasma/plasma_manager.h

```diff
@@ -156,12 +156,27 @@ ClientConnection *ClientConnection_listen(event_loop *loop,
 
 /* Buffer for requests between plasma managers. */
 typedef struct PlasmaRequestBuffer {
+  /** Either PlasmaDataReply or PlasmaDataRequest. */
   int type;
   ray::ObjectID object_id;
+  /**
+   * Shared buffer between manager and store.
+   * Data is read and written to this buffer during
+   * an object transfer.
+   */
   uint8_t *data;
   int64_t data_size;
   uint8_t *metadata;
   int64_t metadata_size;
+  // TODO (hme): Make start/complete an enum.
```

Comment:
Can we do this TODO in this PR? I think that would be nice since the two fields are really describing a single state.